README.rst

```diff
@@ -47,51 +45,35 @@ Get Involved
    kinds of ways to contribute to and interact with the project,
    including mailing list information and how to submit bug reports and
    code to Ansible.
-*  All code submissions are done through pull requests to the ``devel`` branch.
-*  Feel free to talk to us before making larger changes
+*  Join a `Working Group
+   <https://github.com/ansible/community/wiki>`_, an organized community devoted to a specific technology domain or platform.
+*  Submit proposed a code update through a pull request to the ``devel`` branch.
+*  Talk to us before making larger changes
    to avoid duplicate efforts. This not only helps everyone
-   know what's going on, it also helps save time and effort if we decide
+   know what is going on, it also helps save time and effort if we decide
    some changes are needed.
-*  Users list:
-   `ansible-project <https://groups.google.com/group/ansible-project>`_
-*  Development list:
-   `ansible-devel <https://groups.google.com/group/ansible-devel>`_
-*  Announcement list:
-   `ansible-announce <https://groups.google.com/group/ansible-announce>`_
-   -- read only
-*  irc.freenode.net: #ansible
-*  For the full list of Email Lists, IRC channels see the
+*  For a list of email lists, IRC channels and Working Groups, see the
    `Communication page <https://docs.ansible.com/ansible/latest/community/communication.html>`_
 
 Branch Info
 ===========
 
-*  Releases are named after Led Zeppelin songs. (Releases prior to 2.0
-   were named after Van Halen songs.)
-*  The ``devel`` branch corresponds to the release actively under
-   development.
-*  The ``stable-2.x`` branches exist for current releases.
-*  Various release-X.Y branches exist for previous releases.
+*  The ``devel`` branch corresponds to the release actively under development.
+*  The ``stable-2.X`` branches correspond to stable releases.
 *  For information about the active branches see the
    `Ansible release and maintenance <https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html>`_ page.
-*  We'd love to have your contributions, read the `Community
-   Guide <https://docs.ansible.com/ansible/latest/community>`_ for notes on
-   how to get started.
 
 Roadmap
 =======
 
-Based on team and community feedback, an initial roadmap will be published for a major or minor version (ex: 2.0, 2.1).
-Subminor versions will generally not have roadmaps published.
-
+Based on team and community feedback, an initial roadmap will be published for a major or minor version (ex: 2.7, 2.8).
 The `Ansible Roadmap page <https://docs.ansible.com/ansible/devel/roadmap/>`_ details what is planned and how to influence the roadmap.
 
 Authors
 =======
 
 Ansible was created by `Michael DeHaan <https://github.com/mpdehaan>`_
-(michael.dehaan/gmail/com) and has contributions from over 4000 users
-(and growing). Thanks everyone!
+and has contributions from over 4000 users (and growing). Thanks everyone!
```

Comment:
```suggestion
and has contributions from over 4200 users (and growing). Thanks everyone!
```

Reply:
For the sake of long-term maintenance, I'd like to stick with thousands here - if we update for every 100 new contributors, we'll have to check the README pretty often to keep up.

Reply:
@acozine I can write a bot for that :laughing: